lib/src/config/route/routes_generator.dart

```
@@ -23,15 +23,13 @@ class RouteGenerator {
         }
       case initialRoute:
         {
+          return MaterialPageRoute(builder: (context) => HomeScreen());
+        }
+      case movieGridByGenre:
+        if (args is Genre) {
```

`else`? You can either throw an exception or show an error screen for the developer.
(Non-blocker comment 😊 )